tests/test_optimizers.py

```diff
@@ -272,7 +272,54 @@ def test_identity_in_subgraph_non_graph_output(self):
         self.run_identity_compare(["Z1"], {"X": np.random.randn(2, 3, 4, 5).astype(np.float32)},
                                   model_proto, remaining_identity_num=0)
 
-    # Tranpose Optimizer Tests End
+    # Identity Optimizer Tests End
+
+    # Merge Duplicated Nodes Optimizer Tests Start
+
+    def run_merge_duplicated_nodes_compare(self, output_names_with_port, onnx_feed_dict, origin_proto,
+                                           op_type=None, remaining_op_num=None, debug=False, rtol=1e-07):
+        self.run_and_compare(output_names_with_port, onnx_feed_dict, origin_proto, op_type=op_type,
+                             remaining_op_num=remaining_op_num, debug=debug, rtol=rtol)
+
+    def test_duplicated_add(self):
```

Comment:
rename to what we want to test instead of the node name used here.
e.g. test_duplicated_same_inputs?